site/operational-guide.markdown

```diff
@@ -0,0 +1,384 @@
+> fixed) would be to mandate `weave connect` on existing peers as each
+> new peer is added.
+
+Then optionally on each existing peer:
+
+    weave connect <new peer>
+
+This step is not mandatory, but improves the robustness of network
+reformation in the face of node failure as _nodes do not remember to
+connect to discovered peers on restart_. (For example, if you did the
+following:
+
+    host-A$ weave launch; weave consense
+    host-B$ weave launch; weave connect host-A; weave consense
+    host-C$ weave launch; weave connect host-B; weave consense
+
+host-C's ability to connect to host-A would be entirely dependent on
+gossip discovery via host-B; if the whole network rebooted and host-B
+did not come back up then host-A and host-C would not connect to each
+other.)
+
+### Removing a Peer
+
+On peer to be removed:
+
+    weave reset
+
+> Author's Note: releases address space and removes the containers +
+> persistence so that they won't restart on a reboot
+
+On each remaining peer:
+
+    weave forget <removed peer>
+
+> Author's Note: stops reconnection attempts and prevents further
+> connection attempts after restart
+
+## Uniform Fixed Cluster
+
+> Author's Note: I'm going to change this section to recommend systemd
+> at which point the mentions of `weave launch` and `weave connect`
+> will be replaced and I will talk about amending the PEER list in
+> /etc/sysconfig/weave instead
+
+A uniform fixed cluster has the following characteristics:
+
+* Each node has identical configuration
+* There is a controlled process for adding or removing nodes. The end
+  user is responsible for ensuring that only one instance of the
+  process is in-flight at a time
+
+### Bootstrapping
+
+This scenario describes a production deployment of a fixed number of
+N nodes (N=1 in the degenerate case).
+
+> Author's Note: N=1 requires #1881 to be addressed.
+
+On each peer:
+
+    weave launch <initial peer list>
+
+The initial peer list may contain the address of the peer to which it
+is being supplied, so the configuration may be identical for each
+peer.
+
+Then on _any_ peer:
+
+    weave status
+
+To check that all connections have established successfully followed
+by:
+
+    weave consense
+
+To force consensus - future IP address allocations can now continue
+under partition and it is safe to add new peers.
+
+> Author's Note: the rationale for `weave status` is to introduce an
+> opportunity for the user to resolve any initial connectivity
+> problems before consensus - this ensures the ring is divided as
+> evenly as possible, delaying the need for donations
+
+### Adding a Peer
+
+On new peer:
+
+    weave launch <existing peer list>
+    weave consense
+
+On each existing peer:
+
+    weave connect <new peer>
+
+### Removing a Node
+
+On peer to be removed:
+
+    weave reset
+
+On each remaining peer:
+
+    weave forget <removed peer>
+
+## Autoscaling
+
+This configuration builds on an existing fixed cluster (for example of
```

Comment:
I might rephrase this to talk about segregating a fixed cluster of 3-5 machines from the rest of your autoscaling pool.